### PR TITLE
Allow blank software

### LINKED ISSUE #850: Rerunning is not working
Issue raised by: [developer]

### What?
It is not possible to rerun a reduction. When the run is submitted model validation appears to fail because the software field on `ReductionRun` object is blank or null.

### Where?
`WebApp/autoreduce_webapp/reduction_viewer/utils.py`

### How?
Smoke testing

### Reproducible?
[Yes]
click rerun jobs, then submit. If #849 is not merged the exception will be hidden

### How to test the issue is resolved
IT will be possible to rerun jobs in the webapp

### PR DESCRIPTION
**Pull request by keiranjprice101**
### Summary of work
Quick fix to allow reruns will open an issue (#852) to reinforce this and identify why the software was not included originally.

### How to test your work
Verify that a run can be resubmitted in the WebApp

### Additional comments
 

closes #850 

**Before merging ensure the release notes have been updated**

### CODE REVIEW

File: WebApp/autoreduce_webapp/reduction_viewer/models.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ class ReductionRun(models.Model):
                                    on_delete=models.CASCADE)
     retry_run = models.ForeignKey('self', on_delete=models.SET_NULL, null=True, blank=True)
     status = models.ForeignKey(Status, blank=False, related_name='+', on_delete=models.CASCADE)
-    software = models.ForeignKey(Software, blank=False, related_name='reduction_runs', null=True,
+    software = models.ForeignKey(Software, blank=True, related_name='reduction_runs', null=True,
```
Comment (Anders-Markvardsen) [inline WebApp/autoreduce_webapp/reduction_viewer/models.py:116]: A minor. I would suggest that adding a comment above this line saying something like "Allowed software field to be black for now and issued opened (#852) to later this field gets populated properly later".

**Review by JEThompson97 — APPROVED** [review]
So if I understand correctly: this is a quick fix pending a fuller fix (#852)?

**Review by Anders-Markvardsen — APPROVED** [review]
agree looks good

**Comment by Anders-Markvardsen** [comment]
back ported to release branch and dev webapp server updated and restarted

**Pull request merged**